extensions/target-specific/chromium/chromium-helpers/chromium-helpers.js

```diff
@@ -74,6 +84,11 @@ var Chromium = null;
             },
             BrowserProcessEquivalentModules: ["msedge", "accessibility", "chrome"],
 
+            _help_GpuProcessEquivalentModules: {
+                description: "List of modules used in the Chromium gpu process.",
+            },
+            GpuProcessEquivalentModules: ["msedge", "service", "chrome"],
```

Comment:
these should be msedge_child and chrome_child

(can you also verify this works on a non-component build?)